### ISSUE TITLE
Include a feed for syndication

### ISSUE THREAD

**Issue by somini** [nical/nical.github.io#4]
Hi, I wanted to subscribe to your blog but there was no RSS/Atom feed. This is not a static site, so it's not simply toggling a switch.

**Comment by somini** [comment]
My bad, the blog uses Pelican. According to:

https://docs.getpelican.com/en/latest/settings.html?highlight=feed#feed-settings

This should be it:

https://nical.github.io/feeds/all.atom.xml

This is not being updated, though.

**Comment by Mennaruuk** [comment]
https://github.com/nical/nical.github.io/tree/master/feeds

Seems like there've been no updates for a couple of years.

**Comment by nical** [comment]
Thanks for pointing this out I hadn't noticed it stopped working. I expected pelican to generate feeds automatically but it seems that it doesn't or hasn't done it for some time as you said. If I remove the feeds folder from the output directory it isn't generated anymore. I tried creating a new pelican blank site and it doesn't produce feeds either. I also made sure pelican and feedgenerator were installed via pip since there has been some packaging issues with feedgenerator on fedora. I also tried from an ubuntu machine and no feeds are generated either.
I'll keep looking.
